pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -373,6 +373,13 @@ ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 ARG SMD_DATA_PARALLEL_URL
 ARG SMD_MODEL_PARALLEL_URL
 
+# EFA checker binaries
```

Comment:
Can we add a small line in the form of the comment that states  - "Needed for SM Health Checker"?

Reply:
Added